test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -570,3 +621,160 @@ UCS_TEST_P(test_ucp_peer_failure_keepalive, kill_receiver,
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_peer_failure_keepalive)
+
+
+struct test_ucp_peer_failure_rndv_abort : public test_ucp_peer_failure {
+    void init()
+    {
+        self = this;
+
+        if (has_any_transport({"ud_v", "ud_x", "shm,ib", "ugni", "self", "tcp"})) {
+            UCS_TEST_SKIP_R("No rndv/put/zcopy support");
```

Comment:
if we don't skip, would the test fail?

Reply:
can we have better check for this? maybe force RNDV_SCHEME?
also why would the test fail, I would expect that if put/zcopy is not selected there will just not be a simulated failure and test will still pass?

Reply:
`RNDV_SCHEME` is already set, but even with this setting we can fallback to RNDV through AM if RDMA isn't supported. 

The thing is that I especially made the test fail if `rndv/put/zcopy` progress wasn't called to be sure that test actually do what is supposed to do. It checks here:

https://github.com/openucx/ucx/pull/9149/files#diff-02401c91edfb9385548d3b500fe955eaac8d5e625a25c3686c4f4366e84572ceR779

Reply:
so maybe skip the test if the transport is **not** rc or dc?
or check if any of the faces on the worker supports PUT_ZCOPY? or check if we have rma_bw lanes on the ep?

Reply:
> so maybe skip the test if the transport is not rc or dc?

We discussed this option previously with Evgeny and decided to change it to the current form: https://github.com/openucx/ucx/pull/9149#discussion_r1272227363

> or check if any of the faces on the worker supports PUT_ZCOPY? 

I tried this one, but this test also requires check of `UCT_MD_FLAG_INVALIDATE_RMA` and possibly some other flags. So I think that the following option proposed by you looks more laconic. 

> or check if we have rma_bw lanes on the ep?

Applied.